### PR TITLE
Retention Policies licensing required for teams #2904

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving Issue #2904 

Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: Teams/retention-policies-faq.md [hunk 1]
```diff
@@ -38,6 +38,17 @@ Yes, they are. If you create a retention policy to delete data older than 60 day
...
 By default, Teams chat, channel, and files data are retained forever. A user can delete something, but in the absence of retention policies, Teams data is always archived into Exchange online mailboxes (user and group) and stays there for eDiscovery. 
 
+
+### What is the licensing required for retention policy? 
+
+Usually, all those that contain Microsoft Teams and an active Exchange mailbox:
+
+- Office 365 Business Premium
+- Office 365 Business Essentials
+- Office 365 E1, E3 or E5
```
Comment (JohanFreelancer9) [inline Teams/retention-policies-faq.md]:
```suggestion
- Office 365 E1, E3, or E5
```

File: Teams/retention-policies-faq.md [hunk 2]
```diff
@@ -38,6 +38,17 @@ Yes, they are. If you create a retention policy to delete data older than 60 day
 
 By default, Teams chat, channel, and files data are retained forever. A user can delete something, but in the absence of retention policies, Teams data is always archived into Exchange online mailboxes (user and group) and stays there for eDiscovery. 
 
+
+### What is the licensing required for retention policy?
```
Comment (JohanFreelancer9) [inline Teams/retention-policies-faq.md]:
```suggestion
### What is the licensing requirement for a retention policy? 
```

File: Teams/retention-policies-faq.md [hunk 3]
```diff
@@ -38,6 +38,17 @@ Yes, they are. If you create a retention policy to delete data older than 60 day
 
 By default, Teams chat, channel, and files data are retained forever. A user can delete something, but in the absence of retention policies, Teams data is always archived into Exchange online mailboxes (user and group) and stays there for eDiscovery. 
 
+
+### What is the licensing required for retention policy? 
+
+Usually, all those that contain Microsoft Teams and an active Exchange mailbox:
```
Comment (JohanFreelancer9) [inline Teams/retention-policies-faq.md]:
```suggestion
Usually, all those licenses that contain Microsoft Teams and an active Exchange mailbox:
```

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@LolaJacobsen Hi! Following up on this issue, please let me know if there is anything I can do to help!.

**Comment by LolaJacobsen** [comment]
@anwara-ms - please review and let us know whether it's OK to merge this PR.

**Comment by LolaJacobsen** [comment]
Contacting PM Shilpa Ranganathan to review.

**Comment by LolaJacobsen** [comment]
Pinging PM Vasu Rangaswami to review.

**Comment by LolaJacobsen** [comment]
Ping'd PM again this morning.

**Comment by LolaJacobsen** [comment]
Ping'd PM again today.

**Comment by LolaJacobsen** [comment]
After consulting with the PM, we decided that this article is NOT the place for extensive licensing information (which gets out-of-date very quickly). I added a short note about licensing and then linked off to O365 licensing.

**Comment by LolaJacobsen** [comment]
I updated this article in a separate PR. Closing it here.

**Pull request closed without merging**